app/controllers/iiif/auth/v2/probe_service_controller.rb

```diff
@@ -46,6 +43,18 @@ def show # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
           render json:
         end
 
+        def iiif_location(is_geo, file)
+          # 4 hour token
+          token = JWT.encode({ data: 'geo_token', exp: Time.now.to_i + (4 * 3600) }, Settings.geo.proxy_secret, 'HS256')
```

Comment:
This line is happening even for media objects.  Can this be in the `is_geo` conditional?

Reply:
I ended up refactoring this a bit to do this but also not have separate return statements since we no longer need them because video and geo both use a token now